test/testMethods.ts

```diff
@@ -323,7 +323,7 @@ namespace TestMethods {
   export function triggerFakeKeyboardEvent(type: string, target: d3.Selection<void>, keyCode: number, options?: {[key: string]: any}) {
     let event = <KeyboardEvent> document.createEvent("Events");
     event.initEvent(type, true, true);
-    event.keyCode = keyCode;
+    (event as any).keyCode = keyCode;
```

Comment:
i know we've had issues with creating native events (particularly for Phantom), @themadcreator may be able to provide some more context. though if this works then we're good to go!